lib/andomly.rb

```diff
@@ -5,3 +5,9 @@
 module Andomly
   # Your code goes here...
 end
+
+class Array
+  def sample
+    self.include?("nick") ? "nick" : super
```

Comment:
`self.delete` will remove it from the array. I think you want `self.dup.delete("john")`